site/src/examples/file-drop-zone/Error.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactElement } from "react";
+import { Text } from "@salt-ds/core";
+import {
+  FileDropZone,
+  FileDropZoneIcon,
+  FileDropZoneTrigger,
+} from "@salt-ds/lab";
+
+export const Error = (): ReactElement => (
+  <FileDropZone style={{ width: 300 }} status="error">
+    <FileDropZoneIcon status="error" />
+    <strong>File format is not allowed</strong>
```

Comment:
Fer, I changed the title text to "File format is not allowed" here, to align the example to our recommendations. Hope that's ok!

Reply:
thanks! :)